src/components/mobile/molecules/MobileCreateDropdown/MobileCreateDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { CircleClose, CirclePencil } from '@/assets';
+import MobileCreateButton from '@/components/mobile/atoms/MobileCreateButton/MobileCreateButton';
+import { PATH } from '@/constants/path';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useNewSelector } from '@/store';
+import { selectAccessToken } from '@/store/auth';
+import * as S from './MobileCreateDropdown.style';
+
+const MobileCreateDropdown = () => {
+  const navigate = useNavigate();
+  const accessToken = useNewSelector(selectAccessToken);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```
이렇게만 처리되어도 충분한 거 같은데, 혹시 eslint 에러가 나는지 질문드려요! 전체적인 state 에 모두 포함되는 질문입니다!